waitress/task.py

```diff
@@ -766,6 +519,14 @@ def get_environment(self):
                     path = path[len(url_prefix):]
 
         environ = {
+            'REMOTE_ADDR': channel.addr[0],
+            # Nah, we aren't actually going to look up the reverse DNS for
+            # REMOTE_ADDR, but we will happily set this environment variable
+            # for the WSGI application. Spec says we can just set this to
+            # REMOTE_ADDR, so we do.
+            'REMOTE_HOST': channel.addr[0],
```

Comment:
This used to be set after parsing the proxy headers, now it is set before and never updated by the proxy code, `REMOTE_HOST` should match `REMOTE_ADDR` after the proxy code has ran.

Reply:
good catch, I need to fix that